tests/test_repo_legit.py

```diff
@@ -0,0 +1,126 @@
+from os.path import expanduser, isdir, exists, join, abspath, dirname
+from os import makedirs, chmod, walk
+from git import Repo
+import sys
+import legit
+from shutil import rmtree
+import unittest
+
+
+url = 'https://github.com/katridi/test_folder'
+folder = join(expanduser('~'), 'Desktop\\folder2')
```

Comment:
we are not going to depend on github for to test _our_ functionality are we? Or, God forbid, make them OS (Windows) dependent.